persist/fs/files.go

```diff
@@ -881,8 +895,8 @@ func filesetPathFromTime(prefix string, t time.Time, suffix string) string {
 	return path.Join(prefix, name)
 }
 
-func snapshotPathFromTimeAndIndex(prefix string, t time.Time, suffix string, index int) string {
-	name := fmt.Sprintf("%s%s%d%s%s%s%d%s", filesetFilePrefix, separator, t.UnixNano(), separator, suffix, separator, index, fileSuffix)
+func filesetPathFromTimeAndIndex(prefix string, t time.Time, index int, suffix string) string {
+	name := fmt.Sprintf("%s%s%d%s%d%s%s%s", filesetFilePrefix, separator, t.UnixNano(), separator, index, separator, suffix, fileSuffix)
```

Comment:
hey `filesetIndexSegmentFilePathFromTime` below needs to be changed to take a volume index too

Reply:
lol wanted to use the PR so made the change and pushed, hope that's OK :)